shared/uavobjectdefinition/actuatorsettings.xml

```diff
@@ -30,6 +30,9 @@
     <field defaultvalue="0.07" elements="1" limits="%BE:0.0:0.15" name="LowPowerStabilizationMaxPowerAdd" type="float" units="">
       <description>Maximum power to add to ensure stabilization at low power.  Always select a value well under hover power.</description>
     </field>
+    <field defaultvalue="0.3" elements="1" limits="%BE:0.1:1.0" name="LowPowerStabilizationTimeConstant" type="float" units="">
+      <description>Sets the time constant for the power reserve management during low power stabilization.</description>
```

Comment:
Would be nice to at least give an idea of the scale/units so somebody can read this description and then have some idea how to set the value.